examples/ex_openapi/output.gen.go

```diff
@@ -5890,16 +5890,16 @@ func (s *XMLPattern0) init() XMLPattern0 {
 // setDefaults set default value of fields.
 func (s *Schema) setDefaults() {
 	{
-		val := ExclusiveMaximum(false)
-		s.ExclusiveMaximum.SetTo(val)
+		val := bool(false)
+		s.ExclusiveMaximum.SetTo(ExclusiveMaximum(val))
 	}
 	{
-		val := ExclusiveMinimum(false)
-		s.ExclusiveMinimum.SetTo(val)
+		val := bool(false)
+		s.ExclusiveMinimum.SetTo(ExclusiveMinimum(val))
 	}
 	{
-		val := UniqueItems(false)
-		s.UniqueItems.SetTo(val)
+		val := bool(false)
+		s.UniqueItems.SetTo(UniqueItems(val))
 	}
 }
```

Comment:
This is an unintended side-effect of the fix for aliases that have formats. Although it's not as pretty it doesn't have any effect on performance and keeps the template more readable. If however you feel strongly about this I can add some extra handling to the template to avoid it.